### ISSUE TITLE
Delete Container: Aria-required attribute is not defined for "Confirm by typing the container id" mandatory edit field.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1042]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1241070

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section and select any of the item then some new menu item appears in the top like New SQL Query, New Stored Procedure..
Navigate to "Delete Collection" menu item from the more options(...) and select it.
Navigate through all the controls under "Delete  Container" blade.
Navigate to 'Confirm by typing the container ID' required edit field with asterisk mark(*).
Inspect the Edit text box through developer tool F12.
Verify the issue.

Actual Result:
Aria-required attribute is not defined for the "Confirm by typing the container id" mandatory edit field which has an asterisk to symbolize it is required due to which screen reader does not announce as 'required' along with such controls.

Expected Result:
Aria-required attribute should be defined for "Confirm by typing the container id" mandatory edit field which has an asterisk to symbolize it is required so that screen reader will announce 'required' along with such controls.

Note:
This issue exists throughout the application for similar controls like 'Delete table', 'Delete Keyspace' etc.
![image](https://user-images.githubusercontent.com/8519407/132718020-75c76fff-6954-4b97-904e-9a3dcf706c2b.png)
